vyper/parser/context.py

```diff
@@ -15,6 +15,7 @@
 from vyper.signatures.function_signature import (
     VariableRecord,
 )
+from contextlib import contextmanager
```

Comment:
*convention suggestion*

https://github.com/ethereum/snake-charmers-tactical-manual/blob/master/style-guide.md#avoid-from-x-import-y-for-most-standard-library-imports

We have a convention of avoiding `from X import Y` for **most** standard library imports.  There are a few exceptions to this, but we use this as a general rule.  Detailed explanation in the link above, but the short explanation is that when I see `contextmanager` in the code of a module, I need to scroll to the top and find the import to be confident it is what I think it is (or maybe I'm not familiar with contextlib and so I won't even have that intuition).  But when I see `contextlib.contextmanager` it's much easier to reason about without scrolling up to the import.

Reply:
Makes perfect sense, thanks :+1: